driver/modern_bpf/definitions/events_dimensions.h

```diff
@@ -79,5 +79,16 @@
 #define RENAMEAT2_E_SIZE HEADER_LEN
 #define PIPE_E_SIZE HEADER_LEN
 #define PIPE_X_SIZE HEADER_LEN + sizeof(int64_t) * 3 + sizeof(uint64_t) + PARAM_LEN * 4
+#define BPF_E_SIZE HEADER_LEN + sizeof(int64_t) + PARAM_LEN
```

Comment:
I definitely want to find a solution for this file :D

Reply:
yes probably we will add these dims in the event table at the end of the work :thinking: